docs/cli/kops_rolling-update_cluster.md

```diff
@@ -11,8 +11,11 @@ This command updates a kubernetes cluster to match the cloud, and kops specifica
 
 To perform rolling update, you need to update the cloud resources first with "kops update cluster"
 
+Note: terraform users will need run the following commands all from the same directory "kops update cluster --target=terraform" then "terraform plan" then "terraform apply"
```

Comment:
How about:

```Note: terraform users will need to run the following commands prior to running `kops rolling-update cluster`:

* `kops update cluster --target=terraform`
* `cd out/terraform`
* `terraform plan`
* `terraform apply`
```

(it's not all from the same directory, and you want to wrap commands in backticks)